eng/pipelines/corefx-base.yml

```diff
@@ -0,0 +1,134 @@
+parameters:
+  # Required: Name to display for this job in the UI
+  displayName: ''
+  
+  # Required: Name for this specific job -- should be unique per each job.
+  name: ''
+
+  # Required: Pool to run the job on.
+  pool: {}
+
+  # Required: TargetOS value to know what script commands to use
+  # Accepted values:
+  # - Windows_NT
+  # - Linux
+  # - OSX
+  # - FreeBSD
+  targetOS: ''
+
+  # Required: Strategy matrix (the matrix that we're going to build for this job)
+  # The following variables should be defined within the matrix:
+  #   matrix:
+  #     matrix_display_name:
+  #      _configuration: Debug | Release
+  #      _architecture: x64 | x86 | arm | arm64
+  #      framework: (netcoreapp, netfx, uap, etc).
+  #      helixQueues: Windows.Amd64 (Only needed if submitToHelix -> true.) -- Queues should be separated by + if multiple.
```

Comment:
Defaulting this to true *feels* weird.  ie, it seems weird to have to opt out of something (`submitToHelix: false`) rather than opt in (`submitToHelix: true`).  Not a big deal, just seems non-intuitive to me.

Reply:
I can change it. :smile: